herokuappTests/utility/request_utils.py

```diff
@@ -0,0 +1,26 @@
+import requests
+
+class RequestUtils:
+    baseURL = "https://restful-booker.herokuapp.com/"
+
+    def request(self, method, endpoint, **kwargs):
```

Comment:
What is the usage of this method ? From what I see you are directly calling post below.
Idea would be to always call this method from upstream and that method has a handled that handles different type of calls (get, post, put, delete).

Reply:
Also name this method differently or RequestUtils instance upstream so we don't end up with:
`self.request.request(...)`